src/index.tsx

```diff
@@ -1371,4 +1371,26 @@ export class App extends React.Component<any, AppState> {
 const AppWithTrans = withTranslation()(App);
 
 const rootElement = document.getElementById("root");
-ReactDOM.render(<AppWithTrans />, rootElement);
+
+class AppWithErrorBoundary extends React.Component {
+  state = { hasError: false };
+
+  componentDidCatch(error: any, info: any) {
+    localStorage.clear();
+    window.location.reload();
+  }
+
+  static getDerivedStateFromError(error: any) {
+    return { hasError: true };
+  }
+
+  render() {
+    return this.state.hasError ? (
+      <div>Encountered an error. Please reload.</div>
```

Comment:
When I was debugging, without the `location.reload()` above, it always rendered this and immediately afterwards it rendered the App again (and displayed the error stack). Any ideas why?

Reply:
My end goal is to display an error message telling the user to reload (I'd render a reload button, for example). And then if reload didn't work, I'd tell the user to confirm that we'll discard his scene (in localStorage). But I couldn't make the error splash screen (the `<div>`) stay on screen without the error being rethrown (rerendered) FSR. No clue why.

Reply:
I think I got this. This is due to CRA dev mode. In prod mode it would work.